themes/janeswalk/js/components/pages/WalkPage/WalkHeader.jsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+
+//TODO: Define as a path in a common place to share with other components
```

Comment:
This makes more sense, we can do this for commonly used components (i.e. dropdowns) as well. How about for components (or Stores) that are shared?